README.md

```diff
@@ -706,6 +706,19 @@ end
 
 ```
 
+You may wish to have elements declared in a page object class that are not always guaranteed to be present (success or error messages, etc.). If you'd still like to test such a page with `all_there?` you can define an `excluded_elements` method on your page object class that excludes the specified elements from the `all_there?` check.
+
+```ruby
+class TestPage < SitePrism::Page
+  element :name_field, '#name'
```

Comment:
Optional - I'm going to change the docs at some point to use `()` instead of spaces as it's more declarative and avoids any issues. This is going to be a suite wide linting change. If you want to tweak yours here that would be nice.

Reply:
Oh... now I see this.  
I'm afraid that I just suggested the opposite.  😓 Sorry for the confusion @TheMetalCode 

@luke-hill I'm not sure why you want to add the parentheses, but would it make more sense to just do them all at once rather than piece-meal?
The ruby style guide suggests that we should omit parentheses for "Methods that are part of an internal DSL (e.g., Rake, Rails, RSpec)".  I think this applies.
https://github.com/bbatsov/ruby-style-guide#syntax

Reply:
Hmm...I guess I'd say if Rubocop isn't going to enforce parentheses for things like DSL declarations then I'm having a hard time coming up with a reason why we should. My two cents.

Reply:
I've had a read of that article. I was aware of it being the case for things like Rails e.t.c. I guess it's the case that do we declare this is our internal DSL. In which case we probably would. SO happy to be proven wrong here.

There are plenty of instances elsewhere though where we don't use `()` and I will be switching those over. But happy to admit I'm wrong here. 

For reference, at my company we use `()` btu I'm happy for the declared style to omit them. If you both feel as though that is best.

Reply:
I'd probably prefer to leave out the parentheses for the DSL declarations but enforce them for plain-old ruby method arguments and other places where we'd expect them to be present.